webrtc.html

```diff
@@ -5972,8 +5981,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       </div>
       <div>
         <pre class="idl">dictionary RTCRtcpParameters {
-             DOMString cname;
-             boolean   reducedSize;
+             required DOMString cname;
```

Comment:
We don't need these to be marked required, or anything else in RtpParameters. The input validation is handled by comparing against `[[LastReturnedParameters]]`.